lib/task-data/tasks/set-interfaces.js

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
```

Comment:
This file should not be see in your PR since you are not the owner. You need rebase your PR.